src/ex5/dist/clients/itemClient.js

```diff
@@ -0,0 +1,96 @@
+
+
+const URL = "http://localhost:8080";
```

Comment:
you actually don't need this, your server should know to automatically go to localhost when you write the relative path since both your client and server are served from the same express server.